Client/src/Components/NavigationMenu.tsx

```diff
@@ -31,6 +35,15 @@ export default function NavigationMenu() {
   const handleSignIn = () => {
   }
 
+  const history = useHistory();
+
+  function logout() {
+
+    sessionStorage.clear();
```

Comment:
you're not using the function you wrote

Reply:
removeUserInStorage

Reply:
@ds161833 I think we can keep this function for now and use sometime later.

Reply:
no, remove the user from storage do not just clean it up completely, because we may forget that on sign out it cleans up the whole storage and not just the user

Reply:
isn't that what we want? we want top remove the user that's signed in right? @ds161833